requirements.txt

```diff
@@ -1,5 +1,5 @@
 click >= 7.0, < 8.0
-cloudpickle >=0.6.0, <1.5
+cloudpickle >=0.6.0, <1.6
```

Comment:
IMO we should just remove the upper limit on `cloudpickle`. Breaking changes are unlikely, and if needed users are the ones that should be strongly pinning dependencies, not libraries. cc @cicdw for a 👍 / 👎.

Reply:
👍

Reply:
@jcrist I think you should change it to a ceiling on major version, `<2.0`

Reply:
but otherwise I definitely also favor removing the ceiling. Just thought that my be overstepping for me to do as a non-maintainer

Reply:
Sure, that'd match the existing pattern. I still think we should reconsider setting ceilings in the library at all, but that conversation can happen elsewhere.

Reply:
What would you like me to do with this PR? Maybe change it to `< 2.0` for this narrow case, and then you folks can use #2910 to discuss removing version ceilings altogether?

Reply:
Sure, that makes sense to me.

Reply:
Ok cool. I just updated this and changed the title on #2910 , thanks